### DISCUSSION TITLE
butuh batuan : file .env di file yang di share d iGitlab repository tidak ada file .env

### DISCUSSION THREAD

**Discussion by SitiNurbaya28** [teamalgoritma/community#656] [Q&A] [answered]
Tim Algoritma yang Terhormat,

file .env yang saya download dari link yang di share di Gitlab Repository tidak ada file .env nya,,jadi saya buat manual dengan cara membuat folder baru..kemudian saya menyimpan TOKEN di file .env tersebut,tape begitu di run di notebook guide error,,

![Screenshot 2023-05-14 131037](https://github.com/teamalgoritma/community/assets/133105934/23e37c42-4822-4bd4-9768-f4224993ffb5)

![Screenshot 2023-05-14 134923](https://github.com/teamalgoritma/community/assets/133105934/b8d65a49-7b0a-4f97-9c91-4d5f43c9144a)
 ini file .env yang saya buat manual..

**Comment by kevbow** [comment, marked as answer]
Dear @ubay83,
File `.env` yang harus dibuat adalah berupa file text biasa dengan nama `.env` dimana di dalamnya terdapat TOKEN Anda.
Sementara, yang Anda lakukan adalah membuat folder dan di dalamnya terdapat file text bernama TOKEN sehingga tidak berhasil terbaca yang mana merupakan langkah yg kurang tepat.
Semoga membantu.

**Comment by rusdipermana** [comment]
![token](https://github.com/teamalgoritma/community/assets/6158006/adeb06e4-dd03-49d2-b0b0-028be462197f)

tidak usah buat folder baru tapi buat file baru bernama .env

**Comment by nadyadtm** [comment]
Halo @ubay83 , untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah lebih dari 7 hari respon terakhir. Semoga dapat dipahami
